about.html

```diff
@@ -0,0 +1,198 @@
+                                Work with the best designers today! Collaborated with expert designers on our logo. 
+                                Fast & affordable logo design contests. 1M+ samples completed. 
+                                Expert vetted designers. 500,000+ happy designers. Build our entire brand.
+                            </p>
+                            <img src="images/logo.jpeg" alt="logo">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section>
+            <div class="cover-white ">
+                <div class="container">
+                    <div class="past-summits">
+                        <div>
+                            <h2>See past global LIVE AWARE Global SUmmits</h2>
+                            <p class="underline"></p>
+                        </div>
+                        <div>
+                            <p class="about">Take a look at the two global summits that took place onn warshal and brooklyn
+                            </p>
+                            <div class="summits">
+                                <div>
+                                    <div class="text">
+                                        <h3>2013</h3>
+                                        <p>Live ALive Global Summit in warshal</p>
+                                    </div>
+                                    <img src="images/summit1.jpg" alt="logo">
+                                </div>
+                                <div>
+                                    <div class="text">
+                                        <h3>2011</h3>
+                                        <p>Live ALive Global Summit in Brooklyn</p>
+                                    </div>
+                                    <img src="images/summit2.jpeg" alt="logo">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section>
+            <div class="cover-darker desktop-hide">
+                <div class="container">
+                    <div class="partners">
+                        <div>
+                            <h2>partner</h2>
+                            <p class="underline"></p>
+                        </div>
+                        <div>
+                            <img src="images/mozilla.png" alt="mozilla">
+                            <img src="images/google.png" alt="google">
+                            <img src="images/naver.png" alt="naver">
+                            <img src="images/kakao.png" alt="kakao">
+                            <img src="images/bnb.png" alt="airbnb">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
-This section is not present on the about page in the design template, I'd suggest you remove it 👍 👍. Remember it is important that you try to replicate the design template as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Design template:
<img width="480" alt="behance about partners none" src="https://user-images.githubusercontent.com/73607512/198257412-720a65e2-8fdf-4969-97fe-dbb035401be9.PNG">

Reply:
@mckent05 , actually this is part of the about page, but as you recommended, ill have it removed asap.

Reply:
- Hello @SSEKPIUS, I was referring to the 'sponsor' section, that's the part I highlighted in your code 😄. I only attached the picture of the design template as a reference.